rails/project_apis.md

```diff
@@ -126,8 +126,7 @@ This project may seem simple, but now you've got a website that is both a normal
 * [Sophia Wu's solution](https://github.com/SophiaLWu/kittens-api)
 * [Daniel Aguilar's solution](https://github.com/danaguilar/odin-kittens)
 
-
-### Project 2: Using a Third Party API
+## Project 2: Using a Third Party API
```

Comment:
all the headings within a lesson need to use `###`, these are h3's in HTML. They styling on the website will mess up if they are different.